roles/openshift_logging/templates/kibana.j2

```diff
@@ -32,6 +32,16 @@ spec:
           name: "kibana"
           image: {{image}}
           imagePullPolicy: Always
+{% if kibana_memory_limit is not none or kibana_cpu_limit is not none %}
```

Comment:
The specific vars are passed in from the task so you can different values for OPS and non-OPS.  There are a few similar changes in other templates that do the same.